.github/pull_request_template.md

```diff
@@ -1,4 +1,5 @@
 DEVPROD-NNNNN
+<!-- Does this PR have a minor or major release change? Include [minor] or [major] in the title ☝️ -->
```

Comment:
Can we write down and link to some guidance on what exactly constitutes a major or minor change?

Reply:
Since we are using semver, there is already official guidance [documented](https://semver.org/). Since this is new and low stakes, I think we should see how the team uses the tool and go through any discussion that arises from that instead of making a decision framework right away.

Reply:
I created this ticket to come up with some ideas: https://jira.mongodb.org/browse/DEVPROD-11627

Reply:
Agreed Arjun, maybe we should just mention/link to Semver here mostly for the sake of the occasional app engineer opening a PR